runtime/doc/news.txt

```diff
@@ -417,6 +417,10 @@ TREESITTER
   code block fence lines vertically.
 • |vim.treesitter.language.inspect()| shows additional information, including
   parser version for ABI 15 parsers.
+• New |TSQuery:disable_pattern()| and |TSQuery:disable_capture()| to turn off
```

Comment:
```suggestion
• |TSQuery:disable_pattern()| and |TSQuery:disable_capture()| to turn off
```

Reply:
Those links won't work without actual documentation...

Reply:
Just noticed that docs validation isn't working. Working on a fix

Reply:
Ah, I thought it might just fall back to highlighting if there weren't actual docs to link to and I only really added a `news.txt` entry since the check failed on my first push.

Currently it appears there is no documentation at all for `TSQuery` (although it's referenced in some docs), so I'm not sure what the best way to document these new methods would be – would I add a new class type in `query.lua`, or break out a new file in `runtime/lua/vim/treesitter/_meta` or something?

I guess this is related also to #26721 (exposing more of the `TS*` classes in documentation)?

Reply:
Pushed an update which adds `TSQuery` docs to the `lua-treesitter-query` section; it didn't seem to make sense to follow exactly the format already used by `TSNode`/`TSTree` because they have their own sections and there is already a (non-generated) `TREESITTER QUERIES` section, but please let me know if you think it deserves its own section or any wording changes.